user-interface/src/lib/components/Header.tsx

```diff
@@ -41,9 +41,7 @@ export const Header = () => {
   const flags = useFeatureFlags();
   const transferOrdersFlag = flags[TRANSFER_ORDERS_ENABLED];
   const caseSearchFlag = flags[CASE_SEARCH_ENABLED];
-
   const [activeNav, setActiveNav] = useState<NavState>(mapNavState(location.pathname));
-
```

Comment:
I think we would generally have a blank line between the variables and the functions. I'd also be okay with the one above added back in to separate the React state stuff from the regular variables.

Reply:
Added a break above and below. I am going to see if I can enforce line space rules locally so this is consistent